src/plugins/imagery/components/ImageryView.vue

```diff
@@ -1111,7 +1111,7 @@ export default {
       window.clearInterval(this.durationTracker);
     },
     updateDuration() {
-      let currentTime = this.timeContext.clock() && this.timeContext.clock().currentValue();
+      let currentTime = this.timeContext.getClock() && this.timeContext.getClock().currentValue();
```

Comment:
You should no longer need to check for a clock (I think this isn't called if it isn't in realtime anyways), there is always a clock (even in fixed mode now, which should be irrelevant for this logic).